src/components/OrgNode.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="org-node">
+    <input :id="`${prefix}-${id}`" type="checkbox">
+    <li v-bind:class="{ branch: children.length }">
+      <div class="toggle">
+        <label :for="`${prefix}-${id}`">
+          <a :href="`/profile/${data.user_id}`">
+            {{ data.first_name }} {{ data.last_name }}
+          </a>
+        </label>
+        <ul v-for="(child, index) in children" :key="index">
+          <OrgNode :children="child.children" :data="child.data" :id="index" :prefix="`${prefix}-${id}`"></OrgNode>
+        </ul>
+      </div>
+    </li>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'OrgNode',
+  props: {
+    children: Array,
+    data: Object,
+    id: Number,
+    prefix: String,
+  },
+};
+</script>
+
+<style>
+  .org-node label {
+    width: 100%;
+  }
+
+  .org-node label:hover {
+      background-color: lightgray;
```

Comment:
for colours, can we stick to the vars defined in the App component? So this would be: `var(--lightGrey)` (sidenote: we may at some point want to review the greys)